docs/Linking.md

```diff
@@ -2,6 +2,27 @@
 
 ## iOS
 
+#### Project linking
+1. Open your project `.xcodeproj` on xcode.
+
+2. Right click on the Libraries folder and select `Add files to "yourProjectName"`.
+
+3. Add `RNCAsyncStorage.xcodeproj` to your project Libraries.
+
+3. Go to `Build Phases -> Link Binary with Libraries` and add:  `libRNCAsyncStorage.a`.
+
+#### Using 'Pods'
+1. Enter into iOS Folder `cd ios/` (on your project's root folder).
+
+2. Add this line to your `Podfile` just below the last pod: (if you don't one just create a new runnning: `pod init`).
+
+```
+pod 'react-native-async-storage', :path => '../node_modules/@react-native-community/async-storage'
```

Comment:
Let's including diffing, like in Android section:
```
	```diff
	+ pod 'react-native-async-storage', :path => '../node_modules/@react-native-community/async-storage'
	```
```

Reply:
Done :)